packages/server-wallet/src/models/__test__/channel.test.ts

```diff
@@ -1,23 +1,28 @@
+import Knex from 'knex';
+
 import {Channel, ChannelError} from '../channel';
 import {seedAlicesSigningWallet} from '../../db/seeds/1_signing_wallet_seeds';
+import {extractDBConfigFromServerWalletConfig, defaultConfig} from '../../config';
 import {stateWithHashSignedBy} from '../../wallet/__test__/fixtures/states';
-import knex from '../../db/connection';
 
 import {channel} from './fixtures/channel';
 
+const knex: Knex = Knex(extractDBConfigFromServerWalletConfig(defaultConfig));
+
 beforeEach(async () => seedAlicesSigningWallet(knex));
+afterAll(async () => await knex.destroy());
 
 it('can insert Channel instances to, and fetch them from, the database', async () => {
   const vars = [stateWithHashSignedBy()()];
   const c1 = channel({channelNonce: 1234, vars});
 
-  await Channel.query()
+  await Channel.query(knex)
```

Comment:
These changes are quite noisy. An alternative strategy could be
```
import {Channel as UnboundChannel} from 'path/to/channel/model'
const Channel = UnboundChannel.bindKnex(testKnex)
```

If we had a "test" `Channel` model that was bound to the test knex, then the change can be even simpler:
```
import {TestChannel as Channel} from 'path/to/test/channel/model'
```

Reply:
Is this:
```ts
import {Channel as UnboundChannel} from 'path/to/channel/model'
const Channel = UnboundChannel.bindKnex(testKnex)
```
any different to this?
```ts
import {Channel} from 'path/to/channel/model'
Channel.bindKnex(testKnex)
```

Reply:
I don't think that `Channel.bindKnex(testKnex)` has any side effects on the `Channel` class. Instead, it returns a class that is bound to `testKnex`.

https://vincit.github.io/objection.js/api/model/static-methods.html#static-bindknex

```
const BoundModel1 = SomeModel.bindKnex(knex1);
const BoundModel2 = SomeModel.bindKnex(knex2);

// Throws since the knex instance is null.
await SomeModel.query();
```

Reply:
I think the alternative sounds fine, and that pattern is already in use in the e2e test:

https://github.com/statechannels/statechannels/blob/ac5330b40c66c56008e1dae769bdc7359fe1fa60/packages/server-wallet/e2e-test/e2e.test.ts#L35

However, I'm not sure it's worth doing just so we can use `.query()` rather than `.query(knex)`. It also means we have more of a mixture of
i) [`knex` passing](https://vincit.github.io/objection.js/recipes/multitenancy-using-multiple-databases.html#knex-passing-pattern) and
 ii) [model binding]( https://vincit.github.io/objection.js/recipes/multitenancy-using-multiple-databases.html#model-binding-pattern) 
going on: I prefer to keep the same patterns across the codebase if possible, and to have the tests mirror the behaviour of the actual code, too.